engine/installation/linux/oracle.md

```diff
@@ -227,6 +279,11 @@ instead of `yum -y install`, and pointing to the new file.
     $ sudo rm -rf /var/lib/docker
     ```
 
+    > **Note**: This won't work when the `btrfs` graph driver has been used,
+    > because the `rm -rf` command cannot remove the subvolumes that Docker
+    > creates. See the output of `man btrfs-subvolume` for information on
+    > removing `btrfs` subvolumes.
```

Comment:
Add something like, "Subvolumes need to be removed before the rest of the `/var/lib/docker` content can be erased."